include/libdnf5/base/base.hpp

```diff
@@ -79,11 +79,8 @@ class Base {
     /// Returns a pointer to a locked "Base" instance or "nullptr" if no instance is locked.
     static Base * get_locked_base() noexcept;
 
-    /// Call a function that loads the config file, catching errors appropriately
-    void with_config_file_path(std::function<void(const std::string &)> func);
-
-    /// Loads main configuration from file defined by the current configuration.
-    void load_config_from_file();
```

Comment:
We cannot remove those methods. We can mar them deprecated with a reason and then we can remove them. Please also add suggested removal window - like Fedora xx

Reply:
Modified. I recommend removing it as soon as possible. So I wrote in Fedora 40. Or can we do it in Fedora 39?